corehq/apps/toggle_ui/views.py

```diff
@@ -253,8 +253,6 @@ def _call_save_fn_and_clear_cache(static_toggle, previously_enabled, currently_e
 def _clear_caches_for_dynamic_toggle(static_toggle):
     # note: this is rather coupled with python property internals
     DynamicallyPredictablyRandomToggle.randomness.fget.clear(static_toggle)
-    # also have to do this since the toggle itself is cached
-    all_toggles.clear()
```

Comment:
what was broken about this?

Reply:
This function used to use `quickcache` but now it uses `memoized` (https://github.com/dimagi/commcare-hq/commit/9f2ee5f3c999c8089b5d737ac4bc1f4c3982a585#diff-1e75514a2e46b595e721fb25eb941dd5fb43919fde8bc2b9d6c88006319f2de4L428) so there is no cache to clear